website/content/en/docs/overview/project-layout.md

```diff
@@ -0,0 +1,91 @@
+    What is the layout of the projects build with Operator SDK?
+---
+
+## What is the layout of the projects build with Operator SDK?
+
+All projects built with SDK has a common base structure by default which follows the Kubebuilder-style for [What’s in a basic project?][kb-whats in-a-basic-project?] and then, each type has its own specific directories and specific language files.
+
+### Common Base
+
+Following the layout of the common structure.
+
+| File/Directory | Description | 
+| ------ | ----- |
+| `Dockerfile` | It is Dockerfile of your operator project |
+| `Makefile` | It has the helpers to help you work wih your project |
+| `PROJECT` | This file represents the configuration of your project and is used to track useful information for the CLI and plugins |
+| `bin/` | This directory contains useful binaries such as the `manager` which is used to run your project locally and  the `kustomize` one used for the project configuration. For the language types it might have other binaries which help you develop your solution |
+| `bundle/` | This directory is build when you use the Makefile target `make bundle`. You will find here all files which are used to [integrate your project][olm-integrate] with [OLM][olm] with the [bundle][bundle] format. |
+| `bundle/manifests/` | This directory has the [OLM manifests][olm-manifests] of your [bundle][bundle] |
+| `bundle/metadata/`  | This directory has the [OLM metadata][olm-metadata] of your [bundle][bundle]. E.g the index image annotations |
+| `bundle/tests/` | This directory has the [Scorecard][scorecard] tests shipped with your operator bundle. |
+| `config/` | Contains configuration files to launch your project on a cluster. See that plugins might use it to provide their facilities. For example, for the CLI be able to help you out create your operator bundle it will look for the CRD's and CR's which are scaffold in this directory. Also, see that you will find here as well all [Kustomize][Kustomize] YAML definitions. |
+| `config/crd/` | Contains the [Custom Resources Definitions][k8s-crd-doc] |
+| `config/default/` | contains a [Kustomize base][kustomize-base] for launching the controller in a standard configuration. |
+| `config/manager/` | Contains the manifests to launch your operator project as pods on the cluster |
+| `config/manifests/` | Contains the base to generate your OLM manifests in the bundle directory |
+| `config/prometheus/` | Contains the manifests required to enable project to serve metrics to [Prometheus][kb-metrics] |
+| `config/scorecard/` | Contains the manifests required to allow you test your project with [Scorecard][scorecard] |
+| `config/rbac/` | Contains the [RBAC][k8s-rbac] permissions required to run your project |
+| `config/samples/` | Contains the [Custom Resources][k8s-cr-doc] |
+| `hack/` | Contains the file used to scaffold the lincense in the head of your project files ||
+| `bundle.Dockerfile` | The project [bundle][bundle] Dockerfile |
+
+### Ansible
```

Comment:
A quick question. This is similar to the project layout which is explained here: https://sdk.operatorframework.io/docs/building-operators/ansible/reference/scaffolding/.
would it be better to remove that section and point it here ?

Reply:
great point varchar. Let's remove the helm one and keep it centralized and then, indeed remove the duplications and address some nits fixes.